inttest/backup/backup_test.go

```diff
@@ -95,15 +97,17 @@ func (s *BackupSuite) TestK0sGetsUp() {
 	s.Require().NoError(s.StopController(s.ControllerNode(0)))
 	_ = s.StopController(s.ControllerNode(1)) // No error check as k0s might have actually exited since etcd is not really happy
 
-	s.Require().NoError(s.Reset(s.ControllerNode(0)))
-	s.Require().NoError(s.Reset(s.ControllerNode(1)))
+	// Reset will return an error because after starting the controller with --enable-worker
+	// k0s reset will try to delete /var/lib/k0s, which is not possible because it's a volume in docker.
+	s.Reset(s.ControllerNode(0))
+	s.Reset(s.ControllerNode(1))
 
 	s.Require().NoError(s.restoreFunc())
-	s.Require().NoError(s.InitController(0))
+	s.Require().NoError(s.InitController(0, "--enable-worker"))
 	s.Require().NoError(s.WaitJoinAPI(s.ControllerNode(0)))
 
 	// Join the second controller as normally
-	s.Require().NoError(s.InitController(1, token))
+	s.Require().NoError(s.InitController(1, token, "--enable-worker"))
```

Comment:
Just a tiny nit, but I'd place flags first, then the args... 
```suggestion
	s.Require().NoError(s.InitController(1, "--enable-worker", token))
```